packages/modeling/src/geometries/path2/concat.js

```diff
@@ -1,8 +1,9 @@
 const fromPoints = require('./fromPoints')
```

Comment:
For consistent style, let's add this require at the top.

`const vec2 = require('../../maths/vec2')`

Reply:
```js
const { equals } = require('../../maths/vec2')
```